docs/en/faqs/faqs.md

```diff
@@ -51,3 +51,11 @@ The network.endpoint for the Kusama network is `wss://kusama.api.onfinality.io/p
 ## What is the endpoint for the Polkadot mainnet network?
 
 The network.endpoint for the Polkadot network is `wss://polkadot.api.onfinality.io/public-ws`.
+
+## How do I iteratively develop my project schema?
+
+A known issue with developing a changing project schema is that when lauching your Subquery node for testing, the previously indexed blocks will be incompatible with your new schema. In order to iteratively develop schemas the indexed blocks stored in the database must be reset, this can be achieved by launching your node with the `--force-clean` flag. For example: 
```

Comment:
Another thing worth mention is `--force-clean` will let the project indexing from the start block in project manifest, otherwise subql-node will just continue regardless the change of start block.

Reply:
added a small note